examples/Huggingface_Transformers/Pipeline_handler_generalized.py

```diff
@@ -0,0 +1,178 @@
+                "sentiment-analysis":None,
+                "text-classification":None
+            }
+            pass 
+        def __call__(self, data):
+            if self.map_to_postproc_fn[task] is not None:
+                return map(self.map_to_postproc_fn[task], data)
+            else:
+                return data
+    
+    def __init__(self):
+        super().__init__()
+        self.tokenizer = None
+        self.preprocess_factory = self.HFPipelinePreprocessFactory()
+        self.postprocess_factory = self.HFPipelinePostprocessFactory()
+
+
+    def load_model(self, device_id, model_name, hf_models_folder = "/home/model-server/HF-models"):
+        print('Entered `load_model` function')
+        model_folder = os.path.join(hf_models_folder, model_name)
+
+        print("Creating pipeline")
+        pipe = pipeline(task=task, framework=framework, model=model_folder, device = device_id)
+        print("Successfully loaded HF model from folder: {hf_models_folder}/{model_name}")
+        return pipe
+
+    def initialize(self, context):
+        '''
+        context.system_properties['gpu_id'] is decided by Torchserve server to utilize 
+        all available GPUs for inference equally:
+        https://github.com/pytorch/serve/blob/master/docs/custom_service.md#handling-model-execution-on-multiple-gpus
+        '''
+        properties = context.system_properties
+        
+
+        self.map_location = (
+            "cuda"
+            if torch.cuda.is_available() and properties.get("gpu_id") is not None
+            else "cpu"
+        )
+        self.device_id = ( -1 if self.map_location is "cpu" 
+            else int(properties.get("gpu_id"))
+        )
+
+        self.manifest = context.manifest
+
+
+        #Loading model on the 'device' decided by Torchserve
+        #----------------------------------------
+        self.initialized = False
+
+        model_name = context.model_name
+        self.model = self.load_model(self.device_id, model_name)
+
+        self.initialized = True
+        #----------------------------------------
+
+
+    def preprocess(self, data):
+        '''
+        Need to write code to convert the input batch into List[str] that can be processed by the `pipeline` as in this example:
+        https://huggingface.co/spaces/lewtun/twitter-sentiments/blob/main/app.py#L34
+        '''
+
+        #Assuming `data` to be List of txt files, where each txt file contains a single raw inference request input
+
+        print('Preprocessing raw inference requests')
+        data = self.preprocess_factory(data)
+        print('Successfully preprocessed raw inference requests')
+        
+        return list(data)
+
+    def inference(self, data):
+        print(f"Data received by `inference` function is: {data}")
+        preds = self.model(data)
+
+        
+        '''
+        `preds` is something like this for MobileViT XX Small pipeline predictions for 1 inference request:
```

Comment:
put this in docs instead

Reply:
I don't see any place in Docs where a sample output from a specific Huggingface pipeline could be written.

Plus I seem to have missed removing this earlier, should we remove it instead?